### DISCUSSION TITLE
hi Ubuntu22.04_dekstop ipxe menu question

### DISCUSSION THREAD

**Discussion by qsc811022** [ipxe/ipxe#671] [General]
item exit  Exit to BIOS
choose --default linux-net --timeout 10000000000000000000000000000000 option && goto ${option}
:shell
shell
set server_root http: / / 192.168.10.10
:linux-net
kernel  ${server_root}/ubuntuServer/casper/vmlinuz  inst.repo=${server_root}/ubuntuServer initrd=initrd
initrd   ${server_root}/ubuntuServer/casper/initrd
boot
:exit
exit

[
anyone can help me how to slove  IPXE

![88](https://user-images.githubusercontent.com/11502486/177178796-a53c0561-fd06-4f1a-a2eb-a4cef7c6d39b.JPG)
](url)

**Comment by NiKiZe** [comment]
Not sure what the question is.
First I read it as how to exit iPXE, that is simply `exit` but it is up to your bios to handle that as a "boot failure" and attempt next boot devices.
But then comes an image with a linux boot that might have an issue with detecting media.

Now, are you sure that the kernel and initrd supports network boot? and have you maybe tried to run without the initrd to verify that it is loaded correctly (if it is in this case, then removing it would probably give you a different error "earlier")

**Comment by stappersg** [comment]
In #674 raised @qsc811022 the same question.

**Comment by iSentinel** [comment]
try loading the iso image into ram

:ubuntus_2204
set server_root http://${server-ip}/ipxe/Linux/ubuntu/2204/server/
kernel ${server_root}/vmlinuz
initrd ${server_root}/initrd
imgargs vmlinuz initrd=initrd root=/dev/ram0 ramdisk_size=1600000 ip=dhcp url=${server_root}/ubuntu-22.04-live-server-amd64.iso locale=en_US.UTF-8 ro
boot

:ubuntud_2204
set server_root http://${server-ip}/ipxe/Linux/ubuntu/2204/desktop/
kernel ${server_root}/vmlinuz
initrd ${server_root}/initrd
imgargs vmlinuz initrd=initrd root=/dev/ram0 ramdisk_size=3800000 ip=dhcp url=${server_root}/ubuntu-22.04-desktop-amd64.iso locale=en_US.UTF-8 ro
boot

**Reply by qsc811022** [reply]
OK  I try

**Reply by qsc811022** [reply]
@iSentinel  Thanks your  can working   thanks very much

**Reply by iSentinel** [reply]
I'm glad that it turned out) Adjust the size of the ramdisk for yourself. The server version after installation in UEFI does not load itself. You need to enter the bios and choose to boot manually
